sound/soc/sof/ipc4-topology.c

```diff
@@ -486,6 +486,11 @@ static int sof_ipc4_widget_setup_comp_dai(struct snd_sof_widget *swidget)
 		ipc4_copier->copier_config = (uint32_t *)blob;
 		break;
 	}
+	case SOF_DAI_INTEL_SSP:
+		/* set SSP dai index as the node_id */
+		ipc4_copier->data.gtw_cfg.node_id |=
+			SOF_IPC4_INTEL_SSP_NODE_INDEX(ipc4_copier->dai_index);
```

Comment:
Do we need to clear the NODE_INDEX here?

Reply:
i dont think so. it should be 0 at boot and its only assigned once